src/components/templates/ScreenShare/components/Player/Player.scss

```diff
@@ -0,0 +1,50 @@
+.video-player {
+  position: relative;
+  display: flex;
+  border: solid 0;
+  border-radius: 22px;
+  overflow: hidden;
+  width: 480px;
+  height: 320px;
+  margin: 10px;
+
+  .overlay {
+    position: absolute;
+    z-index: 2;
```

Comment:
Please, reuse `z()` css fn from our constants. You can find the examples in `constants.scss`

Reply:
And ensure that we absolutely need to be using a `z-index` in the first place. It's usually a good idea to explain the reasoning/need/etc here in review comments like this.